lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -0,0 +1,265 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass, field
+from pathlib import PurePosixPath
+from time import sleep
```

Comment:
For commit title: do you mean VM *to* VHD?

Reply:
This transformer is to export VHD from a VM. It's expected. Let me know, what's confusing here.

Reply:
You wrote the exact opposite.

Reply:
ok, will change the "as" to "to".